### DISCUSSION TITLE
fortune cookies

### DISCUSSION THREAD

**Discussion by IQGobo** [fornclake/TetraForce#262] [Ideas]
Might refill the looter's health. The main part of this loot is the message contained within, as this could have several uses:

- general hint about the game
- a solution to a (randomized?) puzzle: "Pull the left lever first, then the right one and the middle one last"
- several cookies throughout the world could be required to assemble a code to a lock for an optional dungeon ("The first number is 3", or "The second symbol is the aardvark")
- in randomized games, they could hint about the vague location of progression items
- just something funny, like a joke or just a message trolling the player ("I've slain countless monsters and all I got was this cookie")

Collected fortune cookie messages should be tracked on a secondary screen like the inventory. They should also be shared by all players in a co-op game.

**Comment by IQGobo** [comment]
Fortune cookies can be considered our replacement for sidekicks like Navi or Fi, or calling Ulrira from phone booths.

**Comment by yeslayla** [comment]
Honestly, I could see this super fun if we just hook it up to an array of functions on consume. Who knows what it would do?

A easy way to throw in some easter eggs as well.
